elements/nuxeo-cloud-services/nuxeo-cloud-providers.js

```diff
@@ -292,4 +292,7 @@ Polymer({
       );
     }
   },
+  _editAddLabel() {
```

Comment:
nit: the name of this method tells very little about what it does; this method computes the heading of a dialog, so something like this would be more expressive:
```suggestion
  _dialogTitle() {
```
or
```suggestion
  _computeDialogHeading() {
```